modules/QnA/resources/qnaService.js

```diff
@@ -198,10 +198,15 @@
         };
 
         this.getText = function(){
+           var text = this.cuePoint().text+' ';
+            var reg = /(http:\/\/|https:\/\/).+?\s/gi;
```

Comment:
@MaksymKorniiov  
Please try to run different variations of URLs and see if they are all supported
Try http://www.something.com#abc and http://www.something.com?a=3 etc' and also check how this reacts with spaces.